File: scanner/scanner/models/genre.py
from enum import Enum


class Genre(str, Enum):
	ACTION = "action"
	ADVENTURE = "adventure"
	ANIMATION = "animation"
	COMEDY = "comedy"
	CRIME = "crime"
	DOCUMENTARY = "documentary"
	DRAMA = "drama"
	FAMILY = "family"
	FANTASY = "fantasy"
	HISTORY = "history"
	HORROR = "horror"
	MUSIC = "music"
	MYSTERY = "mystery"
	ROMANCE = "romance"
	SCIENCE_FICTION = "science-fiction"
	THRILLER = "thriller"
	WAR = "war"
	WESTERN = "western"
	KIDS = "kids"
	REALITY = "reality"
	POLITICS = "politics"
	SOAP = "soap"
	TALK = "talk"

	def to_kyoo(self):
		return self.value

File: front/src/models/utils/genre.ts
import z from "zod/v4";

export const Genre = z.enum([
	"action",
	"adventure",
	"animation",
	"comedy",
	"crime",
	"documentary",
	"drama",
	"family",
	"fantasy",
	"history",
	"horror",
	"music",
	"mystery",
	"romance",
	"science-fiction",
	"thriller",
	"war",
	"western",
	"kids",
	"reality",
	"politics",
	"soap",
	"talk",
]);
export type Genre = z.infer<typeof Genre>;
